relay-pii/src/processor.rs

```diff
@@ -753,6 +753,35 @@ mod tests {
         assert_annotated_snapshot!(event);
     }
 
+    #[test]
+    fn test_ignore_user_agent_ip_scrubbing() {
```

Comment:
Another test I would like to see here, if the user would like to scrub the `User-Agent` explicitly now, if that's still possible?

Reply:
So I've looked into this ...

The rule which gets generated from the `DataScrubbingConfig` is this:


```json
"($string || $number || $array || $object) && !(<other ignored fields> || $http.headers.user-agent)": [
  "@common:filter",
  "@ip:replace",
   "strip-fields"
],
```

Where `strip-fields` contains the information gathered from `sensitive_fields` (what the customer can enter in the UI). So the `User-Agent` will never match on the sensitive fields rule.

**But:** with advanced data scrubbing rules the User-Agent can still be scrubbed, since this is an entirely separate process from the `DataScrubbingConfig`.

Should I make a test with `sensitive_fields` set to the user agent and asserting it won't be scrubbed? I feel like this is just a side effect and not something we specifically want to enforce.